.github/workflows/initialize-archiver.yml

```
@@ -9,11 +9,13 @@ on:
         default: ""
         required: true
         type: string
-      sandbox:
-        description: "Initialize a new archive on the Zenodo sandbox server?"
-        default: true
+      server:
+        description: "Which Zenodo server would you like to run on?"
+        options:
+          - sandbox
+          - production
```

Can we have it default to `sandbox` to avoid proliferating drafts on production unnecessarily? It looks like [an available setting](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs).

```suggestion
      server:
        description: "Which Zenodo server would you like to run on?"
        default: 'sandbox'
        options:
          - sandbox
          - production
```